core/src/main/java/io/aiven/klaw/service/UtilControllerService.java

```diff
@@ -63,11 +63,11 @@ public class UtilControllerService {
   @Value("${klaw.coral.enabled:false}")
   private boolean coralEnabled;
 
-  @Value("${klaw.enable.sso:false}")
-  private String ssoEnabled;
+  @Value("${klaw.sso.server.loginurl.azure}")
```

Comment:
No more enable / disable sso?

Reply:
that has become an unused flag, hence removed.